randovania/layout/base/dock_rando_configuration.py

```diff
@@ -215,3 +215,16 @@ def settings_incompatible_with_multiworld(self) -> list[str]:
         if self.mode == DockRandoMode.DOCKS:
             danger.append(f"{self.mode.long_name}: {self.mode.description}")
         return danger
+
+    def dangerous_settings(self) -> list[str]:
+        result = []
+
+        weakness_database = self.weakness_database
+
+        if self.mode == DockRandoMode.WEAKNESSES:
+            for dock_type, state in self.types_state.items():
+                locked = weakness_database.dock_rando_params[dock_type].locked
+                if locked in state.can_change_to:
+                    result.append(f"{locked.name} is unsafe as target for Door Lock Types")
```

Comment:
```suggestion
                    result.append(f"{locked.name} is unsafe as a target in Door Lock Types")
```
Sounds nicer for it IMO, but someone's free to veto me over it

Reply:
Hmmm `in` and `for` both sounds odd to me but I think that's just because it's missing `mode` like `in Door Lock Types mode` or `for Door Lock Types mode`? No idea ¯\\_(ツ)_/¯